src/byte_mapping.rs

```diff
@@ -1,5 +1,30 @@
 const NIL: char = '';
 
+/// The mapping for [ASCII](https://en.wikipedia.org/wiki/ASCII)
+///
+/// This mapping uses the standard ascii character set which is composed of a
+/// 7-bit code (or 128 characters). The first 32 characters and the last one
+/// are non-printable control characters.
+pub const CODEPAGE_ASCII: &'static [char]  = &[
+//   0     1     2     3     4     5     6     7     8     9     A     B     C     D     E     F
+    NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL,  NIL, // 0
```

Comment:
That's why I suggest to use '.', I got a lot of messy code when run the code in different platform with different console/term, if you real need a special symbol, we may use [rust-encoding](https://github.com/lifthrasiir/rust-encoding) for compatibility